package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/refocus-sample-generator-template-utils",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
The version is already 1.0.1 according to https://www.npmjs.com/package/@salesforce/refocus-sample-generator-template-utils, but I think npm publish will bump the version number when we do that, so we shouldn't do it "manually" here

Reply:
When publishing we do - npm version minor - that changes the version here I think. And then npm publish.

Reply:
When publishing we do - npm version minor - that changes the version here I think. And then npm publish.

Reply:
This is from the last time I published this package. It's been sitting in my local changes.

npm publish does not automatically bump the version number.
I think we should check in the updated version here as part of the PR, then publish based on that. If we forget to include it in the first PR and update it in order to publish, we should create a new PR to update the version.

I have updated the version in this PR and opened a new PR for collector-eval to update the version. (I published it just now)

@iamigo @pallavi2209

Reply:
@jgraff2 https://docs.npmjs.com/getting-started/publishing-npm-packages#how-to-update-the-version-number